BraveSharedTests/DAUTests.swift

```diff
@@ -140,6 +140,16 @@ class DAUTests: XCTestCase {
         XCTAssertNil(params2)
     }
     
+    func testMondayOfCurrentWeekFormatted() {
+        // Making sure gregorian year is showing
+        let yearString = Date().mondayOfCurrentWeekFormatted?.truncate(length: 4, trailing: "")
+        let year = Int(yearString!)!
+        
+        // There is no easy way to mock a different calendar in unit tests, just checking if
+        // year we get looks 'correct' should be good enough.
+        XCTAssert(year > 2018 && year < 2025)
```

Comment:
TODO: Come back in 2025 and fix this test 😂

Reply:
I hope they will add calendar setting to test plans in the future 🤡atm there's only language and region